Common Programming Concepts/Repetitions/Print Months/src/main.rs

```diff
@@ -0,0 +1,10 @@
+fn main() {
+    let months = ["January", "February", "March",
+        "April", "May", "June",
+        "July", "August", "September",
+        "October", "November", "December"];
+
+    for month in months {
```

Comment:
I believe this should be 
`for month in months.iter()`
otherwise, the compilation fails

Reply:
Hmm, this raises an interesting question. Which Rust edition and compiler version should we support? The code above works since Rust 1.53 (released on June 17, 2021). Rust got the new "2021" edition since then. Although we mention `edition = "2018"` everywhere, I hadn't spotted the problem because I have a recent `rustc` version installed. Maybe it's a good time to switch to `edition = "2021"`. What do you think?

Reply:
Yes, definitely. Thank you for pointing this out!